Src/CSharpier.MsBuild/build/CSharpier.MsBuild.targets

```diff
@@ -2,7 +2,7 @@
     <PropertyGroup>
         <CSharpier_FrameworkVersion Condition="'$(CSharpier_FrameworkVersion)' == ''">$(TargetFramework)</CSharpier_FrameworkVersion>
         <CSharpierDllPath>$(MSBuildThisFileDirectory)../tools/csharpier/$(CSharpier_FrameworkVersion)/dotnet-csharpier.dll</CSharpierDllPath>
-        <CSharpierArgs Condition="$(CSharpier_Check)">$(CSharpierArgs) --check</CSharpierArgs>
+        <CSharpierArgs Condition="'$(CSharpier_Check)' = 'true'">$(CSharpierArgs) --check</CSharpierArgs>
```

Comment:
I'm thinking this should be an `==` instead? I've always used them.

Reply:
Yes, you are right.
Error when transferring, had already fixed it locally.